h/sql_tasks/tasks/report/create_from_scratch/00_schema/01_create.jinja2.sql

```diff
@@ -1,2 +1,6 @@
 DROP SCHEMA IF EXISTS report CASCADE;
-CREATE SCHEMA report AUTHORIZATION {{db_user}};
+CREATE SCHEMA report AUTHORIZATION "{{db_user}}";
+
+{% for fdw_user in fdw_users %}
```

Comment:
This in being in a loop is a bit extra but:

- Works when the variable is not set (and we get an empty list)
- Works for H where we don't have a a QA environment but we do have `lms-prod` and `lms-qa` users.